android/src/main/java/com/pspdfkit/react/ConfigurationAdapter.java

```diff
@@ -134,6 +139,12 @@ public ConfigurationAdapter(@NonNull Context context, ReadableMap configuration)
             if (configuration.hasKey(SHOW_DOCUMENT_TITLE_OVERLAY)) {
                 configureShowDocumentTitleOverlay(configuration.getBoolean(SHOW_DOCUMENT_TITLE_OVERLAY));
             }
+            if (configuration.hasKey(PAGE_MODE)) {
+                configurePageMode(configuration.getString(PAGE_MODE));
+            }
+            if (configuration.hasKey(FIRST_PAGE_ALWAYS_SINGLE)) {
+                configureFirstPageAlwaysSingle(configuration.getBoolean(FIRST_PAGE_ALWAYS_SINGLE));
```

Comment:
What is the default  of `configuration.getBoolean(...)`. What does it return if the value that was set is not a `boolean`?

Reply:
If it's not a `boolean` an exception is thrown, this makes sense in my opinion so users know if they passed an invalid value in the JS code.